language-support/qute/qute.tmLanguage.json

```diff
@@ -127,9 +133,53 @@
         }
       }
     },
-    "section_start_tag_content": {
-      "match": "((\\s+[^}\\/]+)*)\\s*",
-      "name": "variable.parameter.function.qute"
+    "section_start_user_tag": {
+      "begin": "({)(#)(\\w+(\\.\\w+)*)",
+      "end": "(\\/)?((?<![\\\\])})",
+      "beginCaptures": {
+        "1": {
+          "name": "support.constant.handlebars"
+        },
+        "2": {
+          "name": "keyword.control"
+        },
```

Comment:
Dont use keyword control for user. The hash and the section name must have the same coloris for user znd default tag

Reply:
If the `user` and `default` should have the same colour, then this is already the current behaviour, since `keyword.control` is for the 2nd capturing group, which is `#` and `/`, where the syntax after `#` and `/` are different colours(i.e. `keyword.control` for default and `entity.name.function.identifier` for user, which is the 3rd capturing group). Is this what you wanted?

Reply:
No i want that

#if #with ... have the same color like magenta

#user #iferror have the same color but différent from #if

In other words hash and section name must have the same color

Reply:
Again, this is the current behaviour:
![defaultusersyntaxcolour](https://user-images.githubusercontent.com/26389510/156423760-caba2985-fdfa-44d4-8d6b-e5ca8d40c6b8.gif)

Reply:
# for ifError must have the same color than  ifError

Reply:
Is this better?
![defaultusersyntaxcolour2](https://user-images.githubusercontent.com/26389510/156425459-83a8ef67-1911-42fd-8dc1-3a6f0a13dea7.gif)

Reply:
Yes it's perfect. Thanks @AlexXuChen